src/locales/en/translation.json

```diff
@@ -96,7 +96,20 @@
     "dont_show_it_again": "Don't show it again",
     "copy_to_clipboard": "Copy",
     "copied_to_clipboard": "Copied",
-    "got_it": "Got it"
+    "got_it": "Got it",
+    "repack_language_code": {
```

Comment:
I don't think this is really necessary. Plase check this out:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames
(This applies for all the translations)

Reply:
Perfect, thanks. Done